usecases/traverser/explorer_validate_filters_test.go

```diff
@@ -400,7 +400,7 @@ func Test_Explorer_GetClass_WithFilters(t *testing.T) {
 						search.AssertExpectations(t)
 					})
 
-					t.Run("response must contain concepts", func(t *testing.T) {
+					t.Run("response must contain concepts13", func(t *testing.T) {
```

Comment:
typo?

Reply:
I kept getting errors in those tests and it wasn't reporting which line number it was, so I made them all unique.  I can reformat them to be nicer

Reply:
I don't care too much, just noticed it - will resolve and you can change or not